### PR TITLE
Added textbox

### LINKED ISSUE #10: Transcription view contains a text box
The transcription view should contain a text box. This text box will contain the text found by the TensorFlow model once it has identified all the text in the picture.

### PR DESCRIPTION
**Pull request by AughdonBreslin**
- Put a lil space for a textbox
- Put a lil space for a textbox

closes #10

### CODE REVIEW

File: components/TranscriptionView.tsx [hunk 1]
```diff
@@ -1,8 +1,12 @@
 import React from "react";
-import { View } from "react-native";
+import { View, Text} from "react-native";
 
 const TranscriptionView = (): JSX.Element => {
-	return <View></View>;
+	return (
+		<View>
+			<Text>This is text</Text>
```
Comment (dyc3) [inline components/TranscriptionView.tsx:7]: Can you add a prop for the text content?

**Review by dyc3 — CHANGES_REQUESTED** [review]

**Review by carpenterd777 — CHANGES_REQUESTED** [review]
actual contents of the file look good! it just looks like you gotta run `yarn lint` and commit the result.

**Comment by AughdonBreslin** [comment]
Running yarn lint runs into an error because my computer has disabled running scripts

**Comment by dyc3** [comment]
Can you paste the output for running `yarn lint`?

File: components/TranscriptionView.tsx [hunk 2]
```diff
@@ -1,8 +1,12 @@
 import React from "react";
-import { View } from "react-native";
+import { View, Text} from "react-native";
 
-const TranscriptionView = (): JSX.Element => {
-	return <View></View>;
+const TranscriptionView = (prop:string): JSX.Element => {
```
Comment (dyc3) [inline components/TranscriptionView.tsx:4]: nit, would prefer a name that indicates the prop's purpose, like `transcription`

**Review by dyc3 — DISMISSED** [review]

File: components/TranscriptionView.tsx (hunk 2 above)
Comment (dyc3) [reply components/TranscriptionView.tsx:4]: actually, this doesn't look quite right

**Comment by AughdonBreslin** [comment]
PS C:\Users\aughb\monocle> yarn lint
yarn : File C:\Users\aughb\AppData\Roaming\npm\yarn.ps1 cannot be loaded because running scripts is  
disabled on this system. For more information, see about_Execution_Policies at 
https:/go.microsoft.com/fwlink/?LinkID=135170.
At line:1 char:1
+ yarn lint
+ ~~~~
    + CategoryInfo          : SecurityError: (:) [], PSSecurityException
    + FullyQualifiedErrorId : UnauthorizedAccess

File: components/TranscriptionView.tsx (hunk 2 above)
Comment (AughdonBreslin) [reply components/TranscriptionView.tsx]: it yelled at me that the implied type was 'any' so I put it to a string

**Comment by dyc3** [comment]
See https://stackoverflow.com/questions/4037939/powershell-says-execution-of-scripts-is-disabled-on-this-system

File: components/TranscriptionView.tsx [hunk 3]
```diff
@@ -1,8 +1,12 @@
 import React from "react";
-import { View } from "react-native";
+import {View, Text} from "react-native";
 
-const TranscriptionView = (): JSX.Element => {
-	return <View></View>;
+const TranscriptionView = (transcribedText:string): JSX.Element => {
```
Comment (dyc3) [inline components/TranscriptionView.tsx:4]: The first parameter of a react function component is always an object, called `props` by convention. The object contains all the props. I'd recommend defining an interface for typescript to be happy:

```ts
interface TranscriptionViewProps {
	transcribedText: string;
}
```

And then you would change this line to

```ts
const TranscriptionView = (props:TranscriptionViewProps): JSX.Element => {
```

See the PhotoButton component for an example.

**Review by dyc3 — CHANGES_REQUESTED** [review]

**Comment by AughdonBreslin** [comment]
Changing permissions fixed it, thanks! What is wrong with the prop btw?

File: components/TranscriptionView.tsx (hunk 3 above)
Comment (AughdonBreslin) [reply components/TranscriptionView.tsx:4]: Oh okay, so this way the prop can have multiple attributes?

**Review by carpenterd777 — APPROVED** [review]

**Comment by carpenterd777** [comment]
oh well whoops didn't notice the lint failing, run the linter and *then* I will approve in spirit

**Review by dyc3 — APPROVED** [review]

**Review by dyc3 — APPROVED** [review]
Btw, you don't have to dismiss stale reviews.

**Pull request merged**